src/models/project.js

```diff
@@ -1,5 +1,29 @@
 const { model, Schema } = require('mongoose');
 
+const Comment = new Schema({
+  user: {
+    type: Schema.Types.ObjectId,
+    ref: 'users',
+  }, 
+  commentText : String,
+  },
+  { timestamps: true }
+);
+
+const Review = new Schema({
+  user: {
+    type: Schema.Types.ObjectId,
+    ref: 'users',
+  }, 
+  rating: {
+    type: Number,
+    enum : [1, 2, 3, 4, 5,],
+  },
+  reviewText : String,
+  },
+  { timestamps: true }
+);
+
```

Comment:
good job with adding the embedded schemas in the same project model file and then reference that with type array or comment or review